app/models/reward.rb

```diff
@@ -12,4 +12,19 @@ class Reward < ApplicationRecord
   has_many :categories, through: :category_rewards
 
   has_one_attached :photo
+
+  def self.import(file)
+    ActiveRecord::Base.transaction do
+      CSV.foreach(file.path, headers: true) do |row|
+        if row[3].nil?
```

Comment:
wdyt about using a multi-assignment here? That would increase the code readability :)
<img width="267" alt="Screenshot 2022-05-04 at 15 09 09" src="https://user-images.githubusercontent.com/5034850/166687728-ee36677b-1a36-4883-bc33-ce5f1d970569.png">

Reply:
You're right, it will improve readability. I corrected it